lib/block.mli

```diff
@@ -30,26 +40,68 @@ type section = int * string
 (** The type for sections. *)
 
 (** The type for supported code blocks. *)
-type t = {
+type ocaml_block = {
+  line    : int;
+  file    : string;
+  section : section option;
+  labels  : Label.t list;
+  header  : Header.t option;
+  contents: string list;
+  value   : value;
```

Comment:
All these values are common to all types of blocks. And they make sense on their own (as the "parsed" block), except maybe `value`.

Reply:
done